src/shared/components/Spinner/styles.scss

```diff
@@ -1,17 +1,24 @@
 @import 'shared/styles/colors.scss';
 
+// .spinner {
+//   display: flex;
+//   width: 170px;
+//   height: 170px;
+//   flex-direction: column;
+//   justify-content: space-around;
+//   align-items: center;
+//   align-self: center;
+//   padding-left: 40px;
+//   border: 1px solid red;
+// }
 .spinner {
-  display: inline-block;
-  width: 80px;
-  height: 80px;
-  padding: 25px;
-}
-.spinner:after {
-  content: " ";
-  display: block;
-  width: 64px;
-  height: 64px;
-  margin: 8px;
+  // content: " ";
+  display: flex;
+  align-items: center;
+  align-self: center;
+  width: 54px;
+  height: 54px;
+  margin: 15px;
   border-radius: 50%;
   border: 6px solid green;
   border-color: $green transparent $green transparent;
```

Comment:
Wouldn't `border-color: $green transparent` be enough?

Reply:
Yes, i can removed it. Done